pkg/api/client.go

```diff
@@ -112,6 +118,9 @@ func (c *client) SigningCert(cr CertificateRequest, token string) (*CertificateR
 
 	// Split the cert and the chain
 	certBlock, chainPem := pem.Decode(body)
+	if certBlock == nil {
```

Comment:
FYI, since Decode returns nil for certBlock if it can't find a PEM, I added this check since it certainly seemed like it must return a cert or it should be considered a failure?